provisioner/aws_az_test.go

```diff
@@ -8,10 +8,16 @@ import (
 
 var (
 	info = &AZInfo{
-		subnets: map[string]string{
-			"eu-central-1a": "subnet-1a",
-			"eu-central-1b": "subnet-1b",
-			"eu-central-1c": "subnet-1c",
+		subnets: map[string]SubnetInfo{
+			"eu-central-1a": SubnetInfo{
+				SubnetID: "subnet-1a",
+			},
+			"eu-central-1b": SubnetInfo{
+				SubnetID: "subnet-1b",
+			},
+			"eu-central-1c": SubnetInfo{
+				SubnetID: "subnet-1c",
+			},
```

Comment:
Do we plan to expand this to include tests for `v6` as well?

Reply:
Added a test for the new method introduced here as well